config/locales/help_text/fr.yml

```diff
@@ -16,4 +16,5 @@ fr:
         à votre compte"
       phone: Numéro de téléphone
       social_security_number: Numéro de sécurité sociale
+      verified_at: Date à laquelle vous avez mis à jour ces informations
```

Comment:
Do we have some paid service that provides translations for us?  Or do we just use GoogleTranslate or similar?

Reply:
I think for big batches, yes we have a service. For small lines like this, just sent it through Google Translate